src/builder.rs

```diff
@@ -68,9 +42,24 @@ impl<'a> GlyphBrushBuilder<'a, ()> {
     }
 }
 
-impl<'a, D, H: BuildHasher> GlyphBrushBuilder<'a, D, H> {
+impl<F: Font, D, H: BuildHasher> GlyphBrushBuilder<D, F, H> {
     delegate_glyph_brush_builder_fns!(inner);
 
+    /// When multiple CPU cores are available spread rasterization work across
+    /// all cores.
+    ///
+    /// Significantly reduces worst case latency in multicore environments.
+    ///
+    /// # Platform-specific behaviour
+    ///
+    /// This option has no effect on wasm32.
+    pub fn draw_cache_multithread(mut self, multithread: bool) -> Self {
+        self.inner.draw_cache_builder =
+            self.inner.draw_cache_builder.multithread(multithread);
```

Comment:
Appears to be [true by default](https://docs.rs/glyph_brush_draw_cache/0.1.0/src/glyph_brush_draw_cache/lib.rs.html#201-212); would be good to document that.

Nice to have this addition @alexheretic.